scripts/document.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+if [ -z `which jazzy` ]; then
+    echo "Installing jazzy…"
+    gem install jazzy
+    if [ -z `which jazzy` ]; then
+        echo "Unable to install jazzy."
+        exit 1
+    fi
+fi
+
+WORKSPACE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+cd "${WORKSPACE_DIR}"
+
+jazzy --config "MapboxVision/.jazzy.yaml"
```

Comment:
nit: extract `.jazzy.yaml` string into var and setup `set -u`?

Reply:
Ok